src/components/views/PlayWhites.js

```diff
@@ -4,52 +4,144 @@ import BaseContainer from "components/ui/BaseContainer";
+            }
+            catch (error) {
+                console.error("Details:", error);
+                alert("Invalid Input:\n " + handleError(error));
+            }
+        }
+
+        async function fetchWhiteCards() {
+            try {
+
+                const response = await api.get(`users/${userId}/games/whiteCards`,
+                    {
+                        // reconfiguration might be necessary in case token is not in localStorage here
+                        headers: {
+                            "authorization": token
+                        }
+                    });
+                setCards(response.data)
+                console.log(response.data)
+            }
+            catch (error) {
+                console.error("Details:", error);
+                alert("Invalid Input:\n " + handleError(error));
+            }
+        }
+        fetchGame();
+        fetchWhiteCards();
+    }, [count]);
+
+    // placeholder in case of failure
+    let cardsContent = <div>No white cards available</div>
+    let blackCardContent = <div>No black card available</div>
+
+    if(cards) {
+        cardsContent =
+            <ul className={"game card-list"}>
+                {cards.map(card => (
+                    <WhiteCard card={card} key={card.id}/>
+                ))}
+            </ul>
+    }
+    if(blackCard){
+        blackCardContent =
+            <CardButton className={"card blackCard"}>
+                {blackCard.text}
+            </CardButton>
+    }
+
   return (
     <React.Fragment>
-      <Header view="game"/>
-      <h1>Pick a white card</h1>
-
-      <MenuContainer className={"menu container"}>
-
-        {/* some card */}
-        <CardButton
-          className="card whiteCard"
-          // TODO: onClick: ask to confirm this card, then make api call to submit this card choice, then reload page
-          // onClick={() => history.push('/game/playWhites')}
-          children={"This is a white card"}
-        />
-
-        {/* some card */}
-        <CardButton
-          className="card whiteCard"
-          // TODO: onClick: ask to confirm this card, then make api call to submit this card choice, then reload page
-          // onClick={() => history.push('/game/playWhites')}
-          children={"This is a white card"}
-        />
-
-        {/* some card */}
-        <CardButton
-          className="card whiteCard"
-          // TODO: onClick: ask to confirm this card, then make api call to submit this card choice, then reload page
-          // onClick={() => history.push('/game/playWhites')}
-          children={"This is a white card"}
-        />
-
-
-      </MenuContainer>
+        <Header view="game"/>
+        <div className={"game description"}>
+            <h1>Choose a White Card for the provided Black Card</h1>
+        </div>
+        <div>
+            <button onClick = {() => setCount(count + 1)}>
+                Or Skip this Card </button>
```

Comment:
Do we already allow for skipping cards? And what is the cost for skipping cards?
- if we allow for skipping users will likely only play on Black cards that perfectly suits their white cards which ruins the point of the game.

Reply:
i deleted the skip button